provisioning/service/src/Models/EnrollmentGroup.cs

```diff
@@ -158,7 +158,6 @@ public Attestation Attestation
         /// <summary>
         /// The provisioning status.
         /// </summary>
-        [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
This is probably on the enum directly now, right?

Reply:
Yes, was already tho.